alarms/utils.py

```diff
@@ -36,6 +37,7 @@ def set_config(root_path):
 	parser.add_argument('-L', '--locale', help='Locale for Pokemon names: default en, check locale folder for more options', default='en')
 	parser.add_argument('-d', '--debug', help='Debug Mode', action='store_true')
 	parser.add_argument('-tf', '--time_fix', help='Apply time_fix offset to received times.', action='store_true')
+	parser.add_argument('-ply', '--polygon', help='Only notify if pokemon are within geofence/polygon')
```

Comment:
I like this command, but can we use it to reference a file full of coordinates instead? Seems cleaner then a massive command to launch.